### ISSUE TITLE
Add a global function for audio to find its gain based on the sound settings

### ISSUE THREAD

**Issue by AshHuston** [AshHuston/Tales-of-Andrin-Firetail#116]
I should have settings based on sound preferences (voiceovers, sound effects, ambient, music, etc.) and the function should be called with a type paremeter such as:
gain = getAudioGain("voiceover")
